src/public/partials/header.html

```diff
@@ -1,9 +1,9 @@
 <div class="logo">
   <img src="/images/noun_Tent_2517.svg" alt="tent image for logo" />
-  <a href="index.html"> Sleep<span class="highlight">Outside</span></a>
+  <a href="/index.html"> Sleep<span class="highlight">Outside</span></a>
 </div>
 <div class="cart">
-  <a href="cart/index.html">
+  <a href="/cart/index.html">
```

Comment:
These `/` are very important. Without them the path is relative and so if you click on the logo or the cart icon from product page you're going to go to `https://.../product-pages/index.html` or `https://.../product-pages/cart/index.html`. By prefixing with `/` we make the path absolute so no matter where on the page we click these links we'll go to `https://.../index.html` and `https://.../cart/index.html` as expected.